### ISSUE TITLE
NoSuchFieldException - sHandleToSensor in GyroscopeNoiseFilter.java at 231

### ISSUE THREAD

**Issue by vs5smirnov** [Kjos/JitteryGyroFixForCardboard#11]
Hello, 
Can you please share updated APK  that you posted in this comment: [https://github.com/Kjos/XposedJitteryGyroFix/issues/7#issuecomment-170726380]? That link is no longer working.

I want to try other hooks, because in my case the module fails with NoSuchFieldException for sHandleToSensor field referenced in GyroscopeNoiseFilter.java at 231 (this happens on Cyanogenmod 13).

Thank you in advance!

**Comment by Kjos** [comment]
Hi, sorry to disappoint, but I have abandoned this project a long time ago.
However my pointers would be to compile the source (it was pretty easy), search the Xposed store (I think it may have been removed, but maybe not) or ask lrq3000 since he was involved a lot too. 
There are quite some helpful tutorials and tidbits on developing with Xposed out there if you're looking to develop.

Edit: Just found an apk of this from 12-1-2016. That's all I could find for now. Hope it works.
https://ufile.io/79q68
